addons/website/controllers/manual_models.py

```diff
@@ -0,0 +1,120 @@
+import ast
+
+import werkzeug
+
```

Comment:
since this controller doesn't expose only manual models, it should have another name, the controller class as well

Reply:
Indeed